test/aleph/tcp_ssl_test.clj

```diff
@@ -69,6 +70,31 @@
         (is (nil? @(s/take! c)))
         (is (nil? @ssl-session) "SSL session should be undefined")))))
 
+(def wrong-hostname-cert
+  (SelfSignedCertificate. "some-random-hostname"))
+
+(def wrong-hostname-ssl-server-context
+  (netty/ssl-server-context
+   {:private-key       (.privateKey wrong-hostname-cert)
+    :certificate-chain (.certificate wrong-hostname-cert)}))
```

Comment:
This stuff is being replicated in a couple places...I suggest movign them to `test/aleph/ssl.clj`, where they can be shared

Reply:
Good call. Did that and refactored all SSL tests to use the helpers from that namespace consistently.